### PR TITLE
12.0 mig print attachment wizard

### PR DESCRIPTION
**Pull request by KARIZMA-CONSEIL**

### CODE REVIEW

File: base_report_to_printer/wizards/print_attachment_report.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
```
Comment (i-vyshnevska) [inline base_report_to_printer/wizards/print_attachment_report.xml:2]:
```suggestion
<odoo>
```

File: base_report_to_printer/wizards/print_attachment_report.xml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    <data>
```
Comment (i-vyshnevska) [inline base_report_to_printer/wizards/print_attachment_report.xml:3]:
```suggestion
```

File: base_report_to_printer/wizards/print_attachment_report.xml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            <field name="name">Print Attachments</field>
+            <field name="type">ir.actions.act_window</field>
+            <field name="res_model">wizard.print.attachment</field>
+            <field name="view_type">form</field>
+            <field name="view_mode">form</field>
+            <field name="target">new</field>
+        </record>
+        <menuitem id="menu_action_wizard_print_attachment" action="action_wizard_print_attachment" sequence="50" parent="printing_menu" />
+    </data>
+</openerp>
```
Comment (i-vyshnevska) [inline base_report_to_printer/wizards/print_attachment_report.xml:50]:
```suggestion
</odoo>
```

File: base_report_to_printer/wizards/print_attachment_report.py [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    _name = 'wizard.print.attachment.line'
+    _description = 'Print Attachment line'
+
+    wizard_id = fields.Many2one("wizard.print.attachment")
+    attachment_id = fields.Many2one(
+        'ir.attachment',
+        required=True,
+        domain="['|', ('mimetype', '=', 'application/pdf'), ('mimetype', '=', 'application/octet-stream')]"
+    )
+    record_name = fields.Char(related="attachment_id.res_name", readonly=True)
```
Comment (i-vyshnevska) [inline base_report_to_printer/wizards/print_attachment_report.py:64]:
```suggestion
    record_name = fields.Char(related="attachment_id.res_name")
```

**Comment by yvaucher** [comment]
Superseeded by #222

**Pull request closed without merging**